content/md/user-management/configure-saml-sso.md

```diff
@@ -54,6 +54,10 @@ SAML (Security Assertion Markup Language) is an XML-based standard for exchangin
 
 SAML-based Single Sign-On is a two-way communication between an authentication server (the Identity Provider) and an application (the Service Provider). Therefore, your PIM needs to be configured to declare who will be the authentication server and how the communication must work.
 
+## Pre-Requisites
+
+Before proceeding with the configuration make sure you have configured your **AKENEO_PIM_URL** on your PIM. This variable can be edited in **app/config/pim_parameters.yml**. This variable match the URL you will be accessing the PIM from and will be used to auto-generate the SP ACS and SP Metadata URL.
```

Comment:
Before proceeding with the configuration of the SSO feature, make sure you have configured your **AKENEO_PIM_URL** variable on your PIM. This variable can be edited in **app/config/pim_parameters.yml**. It needs to match the URL you will be accessing the PIM from and will be used to generate automatically the SP ACS and SP Metadata URL.